src/vs/workbench/electron-browser/commands.ts

```diff
@@ -373,15 +376,31 @@ export function registerCommands(): void {
 		win: { primary: void 0 }
 	});
 
-	CommandsRegistry.registerCommand('_workbench.diff', function (accessor: ServicesAccessor, args: [URI, URI, string, string]) {
+	CommandsRegistry.registerCommand('_workbench.diff', function (accessor: ServicesAccessor, args: [URI, URI, string, string, vscode.TextDocumentShowOptions]) {
```

Comment:
it's dangerous to use the vscode-module here (actually every outside the extension host) because these type actually don't exist. for interfaces it's OK because they disappear but using `IEditorOption` would have been better